packages/app/src/subpages/notebookSelector/App.tsx

```
@@ -0,0 +1,120 @@
+          )}
+        >
+          <div>
+            <img src={icon} alt="Citronote" className="h-12 rounded-full" />
+            <LogoType className="h-5 mt-4.5 opacity-90" />
+            <div className="text-xs mt-3 opacity-50"><span className="text-primary font-bold">{t("notebookSelector.betaVersionLabel")}</span> / Version {packageJsn.version}</div>
+          </div>
+          <div>
+            <div className="opacity-95 text-sm hiddena">
+              <Trans i18nKey="notebookSelector.readAboutBeta">
+                使用前に、
+                <a href="" className="text-primary">Citronote 3 ベータテスト版について</a>
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)